tox.ini

```diff
@@ -30,6 +29,14 @@ deps = pytest
 commands =
   pytest {posargs}
 
+[testenv:linting]
```

Comment:
This was returned for local usage, right?

Reply:
Right, I find it convenient to have a tox env for pre-commit as I usually don't have it installed globally.